### PR TITLE
Added Three Templates with ReadMe

### PR DESCRIPTION
**Pull request by rishjai-msft**
Added three templates:

- Enable Replication
- Failover + Reprotect
- Failback + Reprotect

### CODE REVIEW

File: ARM Templates/FailoverReprotect/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+        },
+        "recoveryStagingStorageAccount": {
+            "type": "String",
+            "metadata": {
+                "description": "ARM ID of the storage account to be used to cache replication data in the DR region (for Reprotect operation)."
+            }
+        },
+        "recoveryReplicaDiskAccountType": {
+            "defaultValue": "Standard_LRS",
+            "allowedValues":[
```
Comment (ayfathim) [inline ARM Templates/FailoverReprotect/azuredeploy.json:55]: as per service, the allowed values are:         
        Standard_LRS,

        Premium_LRS,

        StandardSSD_LRS

File: ARM Templates/FailoverReprotect/azuredeploy.parameters.json [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+      "identity": {
+           "value": "GEN-UNIQUE-32",
+          "metadata": {
+              "description": "ARM ID of a user managed identity that has contributor access to the resource group being deployed."
+          }
+      },
+      "vaultSubscriptionId": {
+           "value": "GEN-UNIQUE-32",
```
Comment (ayfathim) [inline ARM Templates/FailoverReprotect/azuredeploy.parameters.json:12]: shouldnt these be blank?

File: ARM Templates/End to End DR/azuredeploy.json [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+            }
+        },
+        "recoveryRegion": {
+            "type": "String",
+            "metadata": {
+                "description": "The designated disaster recovery region where virtual machines would be brought up after failover."
+            }
+        },
+        "policyName": {
+            "defaultValue": "AzureToAzureReplicationPolicy",
```
Comment (ayfathim) [inline ARM Templates/End to End DR/azuredeploy.json:42]: <nit>: the default name is 24-hour-replication-policy used in portal

File: ARM Templates/Enable Replication/azuredeploy.json [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+            }
+        },
+        "policyName": {
+            "defaultValue": "AzureToAzureReplicationPolicy",
+            "type": "String",
+            "metadata": {
+                "description": "Name of the Replication policy to be used to create a new replication policy if protection containers are not mapped."
+            }
+        },
+        "sourceVmARMIdsCSV": {
```
Comment (ayfathim) [inline ARM Templates/Enable Replication/azuredeploy.json:48]: what is the meaning of CSV here in name?

Comment (rishjai-msft) [reply ARM Templates/Enable Replication/azuredeploy.json:48]: Comma Separated Values

File: ARM Templates/End to End DR/azuredeploy.json (hunk 3 above)
Comment (rishjai-msft) [reply ARM Templates/End to End DR/azuredeploy.json:42]: Will update. Thanks.

File: ARM Templates/FailoverReprotect/azuredeploy.parameters.json (hunk 2 above)
Comment (rishjai-msft) [reply ARM Templates/FailoverReprotect/azuredeploy.parameters.json:12]: These are the standard recommended placeholders for ARM Templates.

File: ARM Templates/FailoverReprotect/azuredeploy.json (hunk 1 above)
Comment (rishjai-msft) [reply ARM Templates/FailoverReprotect/azuredeploy.json:55]: Okay. Will update. Thanks!

File: ARM Templates/Enable Replication/README.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
+# Enabling Replication with ASR
+
+This ARM template enables Site Recovery customers to Enable Replication at scale for their Protected Azure VMs.
+
+`Tags: ASR, Site Recovery, Enable Replication`
+
+## Solution overview and deployed resources
+
+Executing this script will deploy a Powershell Script of name "Enable-Replication" and then trigger it. After triggering, Azure VMs will be protected against the Recovery Services Vault.
```
Comment (punit1396) [inline ARM Templates/Enable Replication/README.md:9]: template?

File: ARM Templates/Enable Replication/azuredeploy.json (hunk 4 above)
Comment (punit1396) [reply ARM Templates/Enable Replication/azuredeploy.json:48]: I feel the name need not contain this. The param description already informs the user. Just sourceVmARMIds should suffice. let me know what you think

File: ARM Templates/Enable Replication/README.md (hunk 5 above)
Comment (rishjai-msft) [reply ARM Templates/Enable Replication/README.md:9]: Oops!

File: ARM Templates/Enable Replication/azuredeploy.json (hunk 4 above)
Comment (rishjai-msft) [reply ARM Templates/Enable Replication/azuredeploy.json]: Resolved in the new commit.

File: ARM Templates/Enable Replication/README.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+
+## Solution overview and deployed resources
+
+Executing this template will deploy a Powershell Script of name "Enable-Replication" and then trigger it. After triggering, Azure VMs will be protected against the Recovery Services Vault.
+
+## Prerequisites
+
+Prior to running this ARM template, following things need to be taken care:
+
+- There exists no resource of the name "Enable-Replication" in the resource group
```
Comment (ayfathim) [inline ARM Templates/Enable Replication/README.md:15]: in which resource group

File: ARM Templates/Enable Replication/azuredeploy.json [hunk 7]
```diff
@@ -0,0 +1,121 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "identity": {
+            "type": "String",
+            "metadata": {
+                "description": "ARM ID of a user managed identity that has contributor access to the resource group being deployed."
```
Comment (ayfathim) [inline ARM Templates/Enable Replication/azuredeploy.json:8]: which resource group

File: ARM Templates/Enable Replication/azuredeploy.json [hunk 8]
```diff
@@ -0,0 +1,121 @@
...
+        "recoveryReplicaDiskAccountType": {
+            "defaultValue": "Standard_LRS",
+            "allowedValues":[
+                "Standard_LRS",
+                "Premium_LRS",
+                "StandardSSD_LRS"
+            ],
+            "type": "String",
+            "metadata": {
+                "description": "Type of the Storage account to be used for Disk used for replication."
```
Comment (ayfathim) [inline ARM Templates/Enable Replication/azuredeploy.json:81]: <nit>: small d

File: ARM Templates/Enable Replication/azuredeploy.json [hunk 9]
```diff
@@ -0,0 +1,121 @@
...
+            "identity": {
+                "type": "userAssigned",
+                "userAssignedIdentities": {
+                    "[parameters('identity')]": {}
+                }
+            },
+            "properties": {
+                "forceUpdateTag": "[utcNow()]",
+                "azPowerShellVersion": "3.0",
+                "timeout": "PT4H",
```
Comment (ayfathim) [inline ARM Templates/Enable Replication/azuredeploy.json:113]: azure powershell version 3, why....latest available is 7 naa

**Review by ayfathim — APPROVED** [review]

**Review by punit1396 — APPROVED** [review]

File: ARM Templates/Enable Replication/README.md (hunk 6 above)
Comment (rishjai-msft) [reply ARM Templates/Enable Replication/README.md]: Modified it

File: ARM Templates/Enable Replication/azuredeploy.json (hunk 8 above)
Comment (rishjai-msft) [reply ARM Templates/Enable Replication/azuredeploy.json:81]: Missed it.

File: ARM Templates/Enable Replication/azuredeploy.json (hunk 7 above)
Comment (rishjai-msft) [reply ARM Templates/Enable Replication/azuredeploy.json]: Updated the text.

File: ARM Templates/Enable Replication/azuredeploy.json (hunk 9 above)
Comment (rishjai-msft) [reply ARM Templates/Enable Replication/azuredeploy.json:113]: Not sure, that's how the powershell template has been since the beginning.

**Pull request merged**